test/completion/test_completion_events.vader

```diff
@@ -333,6 +333,27 @@ Execute(b:ale_completion_info should be set up correctly for other sources):
   \ b:ale_completion_info
   Assert !exists('b:ale_completion_result')
 
+Execute(b:ale_completion_info should be set up correctly when requesting completions via callback):
+  let b:ale_completion_result = []
+  call setpos('.', [bufnr(''), 3, 14, 0])
+  function! Callback() abort
+    echo 'Called'
+  endfunction
```

Comment:
Add a `delfunction` line in `After`, or this function will continue to exist when other tests are run. Use a more descriptive name for the test callback, in case it happens to clash with a test function name later.